File: js-quest06/ex00/my_string_index.js
/*
**
** QWASAR.IO -- my_string_index
**
**
** @param {String} param_1
** @param {Character} param_2
** @return {integer}

**
*/


function my_string_index(param_1, param_2) {
    const strLength = param_1.length;
    for(let i = 0; i <= strLength; i++){
        if(param_1[i] === param_2){
            return i;
        }
    }
    return (-1)
};

File: quest02/ex08/my_string_index.c
#include <stdio.h>

/*
**
** QWASAR.IO -- my_string_index
**
** @param {char*} param_1
** @param {char} param_2
**
** @return {int}
**
*/

int my_string_index(char* param_1, char param_2)
{
    int len = strlen(param_1);
    int i;
    for (i = 0; i < len; i++)
    {
        if (param_1[i] == param_2)
        {
            return (i);
        }
    }
    return (-1);
}